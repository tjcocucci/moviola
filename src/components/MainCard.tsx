export function MainCard({ children }: { children: React.ReactNode }) {
  return (
    <div className="lg:pl-72">
      <div className="mx-auto max-w-4xl space-y-8 px-2 pt-20 lg:py-8 lg:px-8">
        {children}
      </div>
    </div>
  );
}
