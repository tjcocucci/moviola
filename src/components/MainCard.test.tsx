import { describe, it, expect } from 'vitest';
import { render } from '@testing-library/react';
import { MainCard } from './MainCard';

const defaultProps = {
  children: <div>test</div>,
};

describe('[ components / MainCard ]', () => {
  describe('when component is rendered', () => {
    it('should render snapshoted MainCard', () => {
      const { asFragment } = render(<MainCard {...defaultProps} />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
