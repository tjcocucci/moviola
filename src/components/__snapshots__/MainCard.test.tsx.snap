// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`[ components / MainCard ] > when component is rendered > should render snapshoted MainCard 1`] = `
<DocumentFragment>
  <div
    class="lg:pl-72"
  >
    <div
      class=" space-y-8  pt-20 lg:py-8 lg:px-8"
    >
      <div>
        test
      </div>
    </div>
  </div>
</DocumentFragment>
`;
